content/assets/blog/phoenix-case-study/index.md

```diff
@@ -0,0 +1,150 @@
+from any device with an internet connection. It allows peacebuilders to collect social media data,
+explore data points, easily configure analyses, and review insights without advanced technical
+skills.
+
+### **Authentication and dashboarding**
+
+Phoenix’s industry standard authentication system ensures secure access, while its Superset-based
+dashboards make data visualization highly flexible. Users can build interactive charts and graphs
+that help surface trends or spikes in conflict-related discussions—providing clarity on community
+sentiment and emerging risks.
+
+### **Data scraping integration**
+
+By integrating with data-scraping platforms like Apify, Phoenix streamlines large-scale data
+collection from social media and other online sources. It lets users define specific parameters for
+what data to capture, helping them focus on the most relevant content while honouring platform and
+data privacy policies.
+
+### **Classification systems for all (Unicode-supported) languages**
+
+A standout feature is Phoenix’s multilingual classification engine, which manages high- and
+low-resource languages alike. This ensures peacebuilders can quickly identify posts of
+interest—crucial when sifting through huge volumes of potentially conflict-related material.
+
+***
+
+## **How it works**
+
+1. **Gather**: Peacebuilders specify keywords, locations, or other criteria for collecting data
+   from social media and other sources.
+
+2. **Classify**: Phoenix’s machine learning models process the incoming data, identifying critical
+   topics, trends, and sentiments.
+
+3. **Explore**: Users start with a standard dashboard and then tailor visualizations to track
+   data trends they are interested.
+
+4. **Collaborate & share**: Being open-source, Phoenix allows contributors worldwide to enhance its
+   capabilities—ensuring continued evolution and impact. As well as increasing the transparency and
+   accessibility of social media analysis for a positive impact.
+
+***
+
+## **The results: A global impact on peacebuilding**
+
+Since launch, Phoenix has proven invaluable for peacebuilders worldwide, yielding several
+noteworthy outcomes:
+
+1. **Global reach**: Versions v1 and v2 of Phoenix have supported hundreds of peacebuilders,
+   allowing them to analyze social media discourses and develop targeted interventions. Including
+   being one of the 31 of 770 projects accepted into the [Paris Peace
+   Forum](https://www.elysee.fr/en/sommet-pour-l-action-sur-l-ia/2024/10/24/call-for-projects-from-the-paris-peace-forum-a-growing-interest-in-addressing-global-challenges-through-ai).
+
+2. **Massive data handling at low cost**: Phoenix has processed terabytes of social media data,
+   managing the scale more commonly associated with large tech firms than small nonprofits. With
+   the platform, that Build Up maintains, being free for small organisations that make a positive
```

Comment:
What do you mean by "that Build Up maintains"? Don't we maintain it? Or do they, because we are on their payroll?

Reply:
de3c20b